CodeConverter/CSharpToVB/ProgressReport.vb

```diff
@@ -0,0 +1,35 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Public Structure ProgressReport
+    Implements IEquatable(Of ProgressReport)
+
+    Public Sub New(Current As Integer, Maximum As Integer)
+        Me.Current = Current
+        Me.Maximum = Maximum
+    End Sub
+
+    Public ReadOnly Property Current As Integer
+    Public ReadOnly Property Maximum As Integer
```

Comment:
This is where any new pieces of progress information would be added, e.g. a status message.

Reply:
I think this is where the sleep delay goes.